src/datasource.ts

```diff
@@ -85,4 +85,16 @@ export class DataSource extends DatasourceWithAsyncBackend<AthenaQuery, AthenaDa
 
     return super.query(options);
   }
+
+  getDefaultQuery(_: CoreApp): Omit<AthenaQuery, 'refId'> {
```

Comment:
just curious do we omit refId because getDefaultQuery has to add it itself directly depending on which query it's adding?

Reply:
Exactly, ref id is "owned" by the query editor rows renderer, not by the plugin. 

Even though omitting ref id is technically fine, I think it's better to type this as `Partial<AthenaQuery>` since that is how it's typed in the datasource api.